src/email-sender.js

```diff
@@ -0,0 +1,58 @@
+const nodemailer = require('nodemailer');
+
+/*
+                             HOW TO USE
+ Import this file - const sendEmail = require('./email-sender);
+ send email - sendEmail.sendMessage('emailName@hotmail.com','Seneca had an error',
+ '<p>An Error had occured</p>');
+
+ Parameters examples
+ receipiants - 'emailName@hotmail.com,emailName2@hotmail.com'
+ subjectMessage - 'Seneca Telescope had an error'
+ message - '<p>An Error has occurred <br/>Error Message:</p>'
+*/
+
+exports.sendMessage = async function (receipiants, subjectMessage, message) {
+  return new Promise((resolve, reject) => {
+    const transporter = nodemailer.createTransport('SMTP', {
+      // Email Server (.env variable must be used refer to the Contribution.md)
+      host: process.env.NODEMAILER_SERVER,
+      port: 25,
+      secure: false,
+      auth: {
+        // Email Name (.env variables must be used refer to the Contribution.md)
+        user: process.env.NODEMAILER_USERNAME,
+        // Email Pass (.env variables must be used refer to the Contribution.md)
+        pass: process.env.NODEMAILER_PASSWORD,
+      },
+    });
+
+    // verify connection configuration
+    transporter.verify((error, success) => {
+      // If error then print to console
+      if (error) {
+        console.log(error);
+      // else print a ready message
+      } else if (success) {
+        console.log('Server is running properly');
+      }
+    });
+
+    // Email Content
+    const mailOptions = {
+      from: process.env.NODEMAILER_USERNAME, // Email Name
+      to: receipiants, // People to send to
+      subject: subjectMessage, // Subject Line
+      html: message, // Message Body
+    };
+
+    // Send the email with the email content
+    transporter.sendMail(mailOptions, (err, info) => {
+      if (err) {
+        reject(err); // Send promise.reject if an error occurs
+      } else {
+        resolve(info); // Send promise.resolve if an error occurs
```

Comment:
One quick thing, could we resolve with info.accepted? That way with our test we can just test if it resolves with the correct recipients?

Reply:
ya for sure, let me include it

Reply:
would you like me to use info.rejected as well for the reject or is the error enough?

Reply:
Ah yes, could you please add that as well? After that I think it should be all good!

Reply:
No, I think it's good actually! It should return the error.

Reply:
okay anything to change, if there isn't we should be good to merge. 
If there isn't anything please approve of these changes, thanks.